app/packages/core/src/components/Actions/similar/utils.ts

```diff
@@ -184,13 +202,10 @@ export const currentSimilarityKeys = selectorFamily<
   get:
     ({ modal, isImageSearch }) =>
     ({ get }) => {
-      const searchBrainKey = get(searchBrainKeyValue);
       const keys = get(availableSimilarityKeys({ modal, isImageSearch }));
-      const result = keys.filter((k) => k.includes(searchBrainKey)).sort();
-
       return {
         total: keys.length,
-        choices: result.slice(0, 11),
+        choices: keys.slice(0, 11),
```

Comment:
I think we need to make all brain keys accessible. This feature will have weird behavior if not.

Reply:
Yes, I agree. Will remove the slice.